parser.py

```diff
@@ -390,30 +432,48 @@ def main():
     # Parse the followings
     following = []
     following_json = read_json_from_js_file(os.path.join(data_folder, 'following.js'))
+    following_ids = []
     for follow in following_json:
         if 'following' in follow and 'accountId' in follow['following']:
-            id = follow['following']['accountId']
-            handle = users[id].handle if id in users else '~unknown~handle~'
-            following.append(handle + ' ' + user_id_URL.format(id))
+            following_ids.append(follow['following']['accountId'])
+    lookup_users(following_ids, users)
```

Comment:
Need to confirm with user before doing any internet access.

Reply:
Added in changeset e6cb01b.